Add database

destinyApproaching opened the pull request.

destinyApproaching committed:
ТЗ11

destinyApproaching committed:
ТЗ11

destinyApproaching committed:
убран пробел

destinyApproaching committed:
убран пробел

destinyApproaching committed:
убран пробел

destinyApproaching committed:
убран пробел

Kapinosva reviewed (commented):
Зря переделал старую реализацию.

сделай просто Impl а остальное разрули аннотациями

Kapinosva reviewed (commented):
Зря переделал старую реализацию.

сделай просто Impl а остальное разрули аннотациями

destinyApproaching committed:
refact: во всех контролерах связка теперь не с реализацией, а с интерфейсом; во всех сервисах связка теперь не с реализацией, а с интерфейсом; в хранилище FilmDbStorage связка теперь не с реализацией, а с интерфейсом; во всех сервисах теперь аннотация @Service; во всех хранилищах теперь аннотация @Repository;
add: добавлен новый метод в интерфейс GenreStorage;
update: метод getFilmById(int id) теперь не получает кучу полей с жанрами;

destinyApproaching committed:
удалён комментарий

destinyApproaching commented:
На GIT`е какая то проблема, он не включает авто-тесты

Kapinosva reviewed (commented):
не прошли чеки и надо сделать меньше обращений к БД

destinyApproaching committed:
Исправлены запросы в БД

destinyApproaching committed:
Исправлены запросы в БД

Kapinosva reviewed (commented):
Сложный код получился. И до идеала еще ест над чем поработать. Надеюсь в следующих работах учтешь
